.coveragerc

```diff
@@ -1,4 +1,11 @@
 # .coveragerc to control coverage.py
+
+# The files in the scripts folder will be deleted once the migration to Docker is complete.
+# Currently, the backend tests for these files are failing when runnning within the Docker container.
+
+# The files in the jobs folder will be removed from here, after upgrading to Python 3.11 or later.
```

Comment:
Put this above the `jobs` block. Include a TODO pointing to the tracking issue on our GitHub for reinstating coverage.